modules/cms-kit/src/Volo.CmsKit.Public.Application/Volo/CmsKit/Public/Blogs/BlogFeaturePublicAppService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.Extensions.Caching.Distributed;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Volo.Abp.Caching;
+using Volo.CmsKit.Blogs;
+
+namespace Volo.CmsKit.Public.Blogs
+{
+    public class BlogFeaturePublicAppService : CmsKitPublicAppServiceBase, IBlogFeaturePublicAppService
+    {
+        protected IBlogFeatureRepository BlogFeatureRepository { get; }
+        protected IDistributedCache<BlogFeatureDto> Cache { get; }
+
+        public BlogFeaturePublicAppService(
+            IBlogFeatureRepository blogFeatureRepository,
+            IDistributedCache<BlogFeatureDto> cache)
+        {
+            BlogFeatureRepository = blogFeatureRepository;
+            Cache = cache;
+        }
+
+        public async Task<BlogFeatureDto> GetAsync(Guid blogId, string featureName)
+        {
+            return await Cache.GetOrAddAsync(
+                $"{blogId}_{featureName}",
+                () => GetFromDatabaseAsync(blogId, featureName),
+                () => new DistributedCacheEntryOptions
+                {
+                    AbsoluteExpirationRelativeToNow = TimeSpan.FromMinutes(1)
```

Comment:
We can use `SlidingExpiration` for cache storage if it is not necessary to deleting one minute later. 
It will remove the cached item after NOT USED for X time

https://docs.abp.io/en/abp/latest/Caching#available-options